base/qaqc/R/cull_database_entries.R

```diff
@@ -15,26 +15,54 @@
 ##' 
 ##' 
 
-cull_database_entries<-function(table, outdir, file_name, con, machine_id = NULL){
+cull_database_entries<-function(table = NULL, outdir, file_name = NULL, con, machine_id = NULL, table_name = NULL){
   
-  file<-paste(outdir,"/",file_name, sep = "")
   
-  table<-read.table(file = file, header = TRUE, sep = "|")
+  if(is.null(table)){
+    
+    if(is.null(file_name)){
+      
+      PEcAn.logger::logger.severe("If a table object hasn't been provided, a file_name must be set.")
+      
+    }else{
+      file<-paste(outdir,"/",file_name, sep = "")
+      table<-read.table(file = file, header = TRUE, sep = "|") # '|' chosen because it's unlikely to overlap with "notes" text. 
```

Comment:
choice of | as delimiter seems odd and risky, plus will be a pain for folks who want to go through the table in a spreadsheet, why not just do csv?

Reply:
I choose | just because when I did "," I got error when I encountered comments with comma's  in them.

Reply:
like 90% sure that was why